CYD-poolen-söker-systemadministratör.md

```diff
@@ -1,20 +1,24 @@
-Är du duktig på Windows, Linux, Solaris eller TCP/IP?
+**1500 unika användare kan inte logga in och 113 000 page requests misslyckas - vem bryr sig?**
```

Comment:
Detta är en sammanslagning av titlarna på affischerna, stating the obvious here but, men dom vi söker har just just läst detta. Då är den gamla titeln med lite andra ämnen bättre. Eller någonting helt nytt. Jag är trött och fortsätter skriva imorgon.